open-api-spec.yaml

```diff
@@ -45,20 +45,20 @@ tags:
   - name: stream-connection
     description: interledger STREAM connections
 paths:
-  '/{accountId}':
+  '/':
```

Comment:
Shouldn't that be `/{paymentPointer}`? And then all the other endpointes `/{paymentPointer}/...`, except connection, which is `/{connection}`?

Reply:
We'd still like the `servers` at the top to represent the payment pointer.
@matdehaast discovered that you can override the `servers` for a particular path, which we can do for `/connections/{connectionId}`